packages/charts/src/chart_types/xy_chart/annotations/line/tooltip.test.tsx

```diff
@@ -97,10 +100,14 @@ describe('Annotation tooltips', () => {
           />
         </Chart>,
       );
-      const annotation = wrapper.find('.echAnnotation__marker');
-      expect(wrapper.find('.echAnnotation')).toHaveLength(0);
-      annotation.simulate('mouseenter');
-      expect(wrapper.find('.echTooltip__header')).toHaveLength(0);
+
+      expect(container.querySelectorAll('.echAnnotation').length).toBe(0);
+      const marker = container.querySelector('.echAnnotation__marker');
+      if (marker) fireEvent.mouseEnter(marker);
+
+      // Tooltip should not show when hideTooltips is true
+      // Use the actual class used by the tooltip header
```

Comment:
This is already in the test name, I think we can remove it
```suggestion
      // Use the actual class used by the tooltip header
```